test/teiserver/protocols/tachyon/v1/tachyon_raw_test.exs

```diff
@@ -71,13 +71,15 @@ defmodule Teiserver.Protocols.V1.TachyonRawTest do
     existing_user = new_user()
     data = %{cmd: "c.auth.register", username: "test_name", email: existing_user.email, password: "password"}
     _tachyon_send(socket, data)
+    :timer.sleep(3000)
```

Comment:
Why a sleep period of 3000?

Reply:
Test weren't passing without timer for me and 3000 worked so I didn't try lowering it. I did some more tests, sometimes it works with as low as 10 but not always. 300 seems to work reliably. Removed in last commit.